xla/backends/gpu/runtime/dynamic_slice_thunk.cc

```diff
@@ -63,8 +80,12 @@ DynamicSliceThunk::DynamicSliceThunk(
       offsets_(offsets),
       orig_shapes_(orig_shapes),
       sliced_shapes_(sliced_shapes),
-      offset_byte_sizes_(offset_byte_sizes) {
-  // Zip all arguments together to create a list of SliceDef.
+      offset_byte_sizes_(offset_byte_sizes),
+      extracted_offset_modules_(std::move(extracted_offset_modules)),
+      indvar_init_(std::move(indvar_init)),
+      indvar_update_(
+          std::move(indvar_update)) {  // Zip all arguments together to create a
```

Comment:
Why is there a linefeed here? Seems like the comment should come on the line below :)